prov/psm2/src/psmx2_am.c

```diff
@@ -291,7 +291,8 @@ int psmx2_am_init(struct psmx2_trx_ctxt *trx_ctxt)
 			psmx2_unlock(&psmx2_am_global.lock, 1);
 			FI_WARN(&psmx2_prov, FI_LOG_CORE,
 				"number of PSM2 endpoints exceed limit %d.\n"
-				"at indecies %d, %d\n", PSMX2_AM_MAX_TRX_CTXT);
+				"at indecies %d\n", PSMX2_AM_MAX_TRX_CTXT,
+				psmx2_am_global.cnt);
```

Comment:
We should just remove "at indecies %d, %d\n" here.

Reply:
Thanks!
Fixed